### DISCUSSION TITLE
How to get galaxy braing achivement

### DISCUSSION THREAD

**Discussion by YJPark0421** [community/community#18725] [Other Feature Feedback, Questions, & Ideas] [answered]
https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

![image](https://user-images.githubusercontent.com/88306533/174039176-ba87c052-8438-4c80-9d1a-11046098beb7.png)

**Comment by JoshuaLuckers** [comment, marked as answer]
If you participate in a discussion and someone marks your answer as "the answer"

**Reply by YJPark0421** [reply]
thank you. :)
